src/app/settings/page.tsx

```diff
@@ -26,7 +28,9 @@ export default function Settings() {
         <div>
           <h2 className={styles.SectionLabel}>Account</h2>
           <div className={styles.EditAccountContainer}>
-            <ActiveButton text="Edit Account" />
+            <Link href="settings/edit-email" passHref>
+              <ActiveButton text="Edit Account" />
+            </Link>
```

Comment:
forgot to tell u this in class but rename both the folder and href from `edit-email` to `edit-account`.